server/routines/game.js

```diff
@@ -71,11 +71,19 @@ function getAvailableActions(game, derivedState) {
 		return actions
 	}
 
-	// TODO - add more conditions (e.g. you can't change active hermit if there is only one.
-	// or you can't add more hermits if all rows are filled)
-	actions.push('ADD_HERMIT')
+	let hermits = 0
+	for (let i = 0; i < currentPlayer.board.rows.length; i++) {
```

Comment:
For this I think we can stick to `currentPlayer.board.rows.filter(row => row.hermitCard).length`

Reply:
Fair enough